javascript/new-src/src/js/app.js

```diff
@@ -14,665 +14,702 @@
-      return $http.get('/json/universe_plugin_list').then(function (response) {
-       return response.data;
-      });
-     },
-     ServerInfo: function () {
-      return $http.get('/json/server_stats').then(function (response) {
-       return response.data;
-      });
-     },
-     Ports: function () {
-      return $http.get('/json/get_ports').then(function (response) {
-       return response.data;
-      });
-     },
-     PortsId: function (id) {
-      return $http.get('/json/get_ports?id=' +
-      id).then(function (response) {
-       return response.data;
-      });
-     },
-     InfoPlugin: function (id) {
-      return $http.get('/json/plugin_info?id=' +
-      id).then(function (response) {
-       return response.data;
+  ])
+  .controller('menuCtrl', ['$scope', '$ola', '$interval', '$location',
+    function($scope, $ola, $interval, $location) {
+      'use strict';
+      $scope.Items = {};
+      $scope.Info = {};
+      $scope.goTo = function(url) {
+        $location.path(url);
+      };
+      $ola.get.ItemList().then(function(data) {
+        $scope.Items = data;
       });
-     },
-     Dmx: function (id) {
-      return $http.get('/get_dmx?u=' +
-      id).then(function (response) {
-       return response.data;
+      $ola.get.ServerInfo().then(function(data) {
+        $scope.Info = data;
+        document.title = data.instance_name + ' - ' + data.ip;
       });
-     },
-     UniverseInfo: function (id) {
-      return $http.get('/json/universe_info?id=' +
-      id).then(function (response) {
-       return response.data;
+      $interval(function() {
+        $ola.get.ItemList().then(function(data) {
+          $scope.Items = data;
+        });
+        $ola.get.ServerInfo().then(function(data) {
+          $scope.Info = data;
+        });
+      }, 10000);
+    }])
+  .controller('overviewCtrl', ['$scope', '$ola', '$location',
+    function($scope, $ola, $location) {
+      'use strict';
+      $scope.Info = {};
+      $scope.Universes = {};
+      $ola.get.ItemList().then(function(data) {
+        $scope.Universes = data.universes;
       });
-     }
-    },
-    post: {
-     ModifyUniverse: function (data) {
-      return $http({
-       method: 'POST',
-       url: '/modify_universe',
-       data: postEncode(data),
-       headers: {
-        'Content-Type': 'application/x-www-form-urlencoded'
-       }
-      }).then(function (response) {
-       return response.data;
+      $ola.get.ServerInfo().then(function(data) {
+        $scope.Info = data;
       });
-     },
-     AddUniverse: function (data) {
-      return $http({
-       method: 'POST',
-       url: '/new_universe',
-       data: postEncode(data),
-       headers: {
-        'Content-Type': 'application/x-www-form-urlencoded'
-       }
-      }).then(function (response) {
-       return response.data;
+      $ola.get.Debug().then(function(data) {
+        var regex = /config-dir: (.*)/;
+        $scope.Info.config_dir = regex.exec(data)[1];
```

Comment:
@daveol it must already be exposed via other means, so can we do that rather than use /debug please.